pkg/operator/controller/controller_dns_service_test.go

```diff
@@ -152,3 +162,113 @@ func TestDNSServiceChanged(t *testing.T) {
 		}
 	}
 }
+
+func Test_shouldEnableTopologyAwareHints(t *testing.T) {
```

Comment:
Looking at the code coverage for `shouldEnableTopologyAwareHints` when this test is run I don't see test cases that cover:
- nodeHasExcludedLabels returns true
- !nodeIsReady
- node.Status.Allocatable.Cpu().IsZero() returns true

Reply:
https://github.com/openshift/cluster-dns-operator/compare/a343ca809da3079230527d06bf1815add9cb4afd..4a7037f7704ac2b0a44f9e22d3b28e42b24db5a6 adds test cases for those three conditions.